testsuite/features/srv_delete_channel_npn.feature

```diff
@@ -0,0 +1,79 @@
+# COPYRIGHT (c) 2017 SUSE LLC
+# License under the terms of the MIT License.
+
+Feature: delete channels
```

Comment:
Could we give a feature name which described it a bit more what it test?
About the feature name, what `_npn` means?  Don't worry to have a longer name if need :)

Reply:
thanks for the review.
I add more information to feature name. Regarding the _npn, I had copy this test from another that had that in the name and forget to remove it (I also don't know what that means). I had asked to MC, which I think was who name the file.

Reply:
the npn sufix was just a temporary name for a new test file.